CMakeLists.txt

```diff
@@ -78,7 +78,8 @@ FetchContent_MakeAvailable(repo-common repo-core repo-backend)
 FetchContent_Declare(
   pybind11
   GIT_REPOSITORY "https://github.com/pybind/pybind11"
-  GIT_TAG "v2.10"
+  # COMMIT ID for v2.10.0
+  GIT_TAG "aa304c9c7d725ffb9d10af08a3b34cb372307020"
```

Comment:
If it is resolved in the later version perhaps we can upgrade to that version instead of pinning an older version.

Reply:
AFAIK - the resolution is in master only and not in a tagged release yet.